src/sidebar/components/Menu.js

```diff
@@ -41,16 +41,22 @@ let ignoreNextClick = false;
  *   `false`, the consumer is responsible for positioning.
  * @prop {string} [contentClass] - Additional CSS classes to apply to the menu.
  * @prop {boolean} [defaultOpen] - Whether the menu is open or closed when initially rendered.
+ *   Ignored if `open` is present.
```

Comment:
The changes to `Menu` to enable external control of open/closed state are straightforward, however, I think the API is starting to feel a little confusing. I tried to extend a few comments here but I'm not sure if it's clear enough...

Reply:
This pattern of props to allow for the component to maintain its own state or have it externally controlled is similar to `<input>` in React, which has `value`, `onInput` and `defaultValue` props that work the same way as `open`, `onOpenChanged` and `defaultOpen` here. On that basis I hope it should feel familiar to future devs.